.buildkite/steps/publish.sh

```diff
@@ -153,14 +153,16 @@ echo "--- Building and publishing stack"
 
 make setup build
 
-if [[ $BUILDKITE_BRANCH == "master" ]] ; then
+# Publish the top-level mappings only on version tagged releases
+if [[ "$BUILDKITE_TAG" =~ ^v ]] ; then
```

Comment:
Nope! Not really. Without making it a manual button. Or checking with the GitHub API that it is in fact the "latest"